src/test/java/com/amazon/opendistroforelasticsearch/sql/util/MatcherUtils.java

```diff
@@ -79,4 +84,24 @@ protected U featureValueOf(T actual) {
         return (Matcher) hasEntry(key, value);
     }
 
+    @SuppressWarnings("unchecked")
+    public static Matcher<JSONObject> hitAny(Matcher<? super Object>... matcher) {
```

Comment:
np: isn't `Matcher<?>` enough? I think `super Object` is always implied.

Reply:
Done.